_literate/EX-wine/tutorial.jl

```diff
@@ -170,7 +169,7 @@ PCA = @load PCA
 pca_pipe = Standardizer() |> PCA(maxoutdim=2)
 pca = machine(pca_pipe, Xtrain)
 fit!(pca)
-W = transform(pca, Xtrain)
+W = MLJ.transform(pca, Xtrain)
```

Comment:
Why do we need this `MLJ.` qualification?

Reply:
Tried removing it now and this is what I get
```
WARNING: both DataFrames and MLJ export "transform"; uses of it in module Main must be qualified
ERROR: UndefVarError: `transform` not defined
Stacktrace:
 [1] top-level scope
   @ ~/Documents/GitHub/DataScienceTutorials.jl/_literate/EX-wine/tutorial.jl:172
```

Reply:
It's possible that before activate a new Julia environment and running a new file that I have to exit Julia REPL altogether and then activate the new environment?

Reply:
Now when I tried again after exiting the REPL and starting it again I no longer get the error so this has to be it.

Reply:
Yes, I think the problem is you had code loaded but thought an environment change should have "removed" it. You can't really undo function definitions in Julia, only overload them. You need to restart your session to remove code.